packages/crayons-core/src/components/icon/readme.md

```diff
@@ -1,18 +1,25 @@
 # Icon (fw-icon)
-fw-icon displays an icon-sized image that imparts meaning to the component it is associated with.
 
-## Demo
+"fw-icon/FwIcon" is a renderer of SVG file that displays an icon-sized image that imparts meaning to the HTML component it is associated with.
+Following features are available as part of the implementation of the fw-icon.
+
+1. Enable fw-icon as an optimized renderer for SVG with built-in functions such as Intersection-Observer and Fetch-API Memoization.
+   Go through the docs to understand the various props it supports.
+2. Providing icon-support for crayons-system components and also exposing crayons-icon set for public use with inbuilt support for external icon-libraries also.
+3. Icons are accessibility tree compliant. You need to pass the label attribute. aria-hidden is set to true.
```

Comment:
You have set `aria-hidden` to `undefined` above. Here it is mentioned it as being set to `true`. Can you pls check .

Reply:
ok

Reply:
Fixed